chrome-extensions.md

```diff
@@ -92,4 +92,8 @@
 [WhatFont Extension](https://chrome.google.com/webstore/detail/whatfont/jabopobgcpjmedljpbcaablpmlmfcogm) | What is the easiest way to find out the fonts used in a webpage? Firebug and Webkit Inspector are easy enough to use for developers. It also detects the services used for serving the web fonts. Supports Typekit and Google Font API.
 [Wappalyzer](https://chrome.google.com/webstore/detail/wappalyzer-technology-pro/gppongmhjkpfnbhagpmjfkannfbllamg) | Wappalyzer is an effective tool to help find out all the underlying technologies used to build any website
 [Honey: Automatic Coupons & Rewards](https://chrome.google.com/webstore/detail/honey-automatic-coupons-r/bmnlcjabgnpnenekpadlanbbkooimhnj) | Add Honey to your browser in seconds and shop like normal. We’ll pop up and do the work for you. Save money and earn rewards when you shop online. 
-[GitHub Web IDE](https://chrome.google.com/webstore/detail/github-web-ide/adjiklnjodbiaioggfpbpkhbfcnhgkfe?hl=en-US) | This extension adds to any GitHub repository a new drop down menu which offers links to a number of online services that allow viewing the source code of the repository in an IDE like interface.
\ No newline at end of file
+[GitHub Web IDE](https://chrome.google.com/webstore/detail/github-web-ide/adjiklnjodbiaioggfpbpkhbfcnhgkfe?hl=en-US) | This extension adds to any GitHub repository a new drop down menu which offers links to a number of online services that allow viewing the source code of the repository in an IDE like interface.
+[Visbug] (https://chrome.google.com/webstore/detail/visbug/cdockenadnadldjbbgcallicgledbeoc) | An Open Source design tool, VisBug helps you redesign the UI of any website real-time, giving you the ability to check out what looks good without the need to repeatedly updated the code and refresh the page
+[Wappalyzer](https://chrome.google.com/webstore/detail/wappalyzer-technology-pro/gppongmhjkpfnbhagpmjfkannfbllamg) | Wappalyzer is an effective tool to help find out all the underlying technologies used to build any website.
+[Colorzilla](https://chrome.google.com/webstore/detail/colorzilla/bhlhnicpbhignbdhedgjhgdocnmhomnp)| Colorzilla is an eyedropper tool with a color picker, color history, CSS gradient analyzer, and every other color-related tool you can think of. It is a highly useful Chrome Extension for Web Developers to directly pick up the color and use it in their design or work.
+[check-my-links](https://chrome.google.com/webstore/detail/check-my-links/ojkcdipcgfaekbeaelaapakgnjflfglf)| Just as the name suggests, Check My Link crawls through your web page to show if it has broken links and checks for valid and redirected links It can be easily used and can be very helpful for maintaining the SEO standards of your webpage.
```

Comment:
all these tools were already added. Could you remove these changes, please?

Reply:
Okay, I'll remove them!

Reply:
taking a look on the list of tools (pressing ctrl + F and search for it)